docs/src/digitising-process.md

```diff
@@ -1,184 +1,349 @@
+   ```
+
+   The new CRS was made to save having to datum transformations in the future. It was then decided that
+   georeferencing would be done using the GAUL dataset for reference points because a graticule
+   transformation was not possible.
+
+**Note:** Sample QGIS projects will be provided for users to explore and analyze. The GAUL dataset, as this was the 
+suggested reference dataset. The dataset is available from [here](https://geonode.wfp.org/geoserver/wfs?format_options=charset%3AUTF-8&typename=geonode%3Aadm1_gaul_2015&outputFormat=SHAPE-ZIP&version=1.0.0&service=WFS&request=GetFeature)
 
-The new CRS was made to save having to datum transformations in the future. It was then decided that georeferencing would be done using the GAUL dataset for reference points because a graticule transformation was not possible.
 
 ### Georeferencing Mozambique Southern Region
 
->Multiple iterations were required as the georeferencing was done against a reference dataset and could not be done by simply taking the corner graticules from the tifs and projecting them into the custom CRS.
+**Note:** Multiple iterations were required as the georeferencing was done against a reference dataset and could not
+be done by simply taking the corner graticules from the geotiff and projecting them into the custom CRS.
+
 
-1. The first iteration of georeferencing the Southern Region of Mozambique was done using a linear transformation. The resulting image ended up being too different to the reference dataset spatially and so was immediately discarded.
-![Mozambique South Attempt 1](moz_img/MozSouth_attempt1.png)
+#### Georefencing Parameters for Southern sheet
+Georeferencing is an iterative process, and we needed to try different parameters to establish the most suitable ones 
+to use.
 
-2. The second iteration was done using a Helmert transformation and gave a decent result but there were far too many discrepancies between the reference layer and the georeferenced image.
+The methods below describe the different parameters that were tested with the raster images.
 
-3. The third iteration was done using the Helmert Transform again but with all Residual pixels for the Ground Control Points (CPs) being under 10. 9 GCPS were used for the referencing. There were too many discrepancies between the reference dataset and the georeferenced image so it was disregarded as a viable image.
+##### Test Parameters
+This describes the parameters that were tested, and we found to be not adequate to use for georeferencing.
 
-4. The fourth iteration was done using a Polynomial 1 Transformation (with all residual pixels less than 10) and 12 GCPs. Again, there were too many discrepancies between the reference dataset and the georeferenced image but there was minimal warping on the polygons.
+1. The first iteration of georeferencing the Southern Region of Mozambique was done using a linear transformation.
```

Comment:
These test parameters can we rather have a table like

| Iteration  | transformation type|
| ------------- | ------------- |
| Content Cell  | Content Cell  |
| Content Cell  | Content Cell  |

Reply:
![image](https://user-images.githubusercontent.com/100694811/191520307-57d496bb-6dc4-48f6-a7b5-3473d951aefe.png)